.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
-## Checklist
-- [ ] Reference related [issue](https://github.com/vectorizedio/redpanda/issues)
-- [ ] Update [PendingReleaseNotes.md](https://github.com/dotnwat/redpanda/blob/dev/PendingReleaseNotes.md), if relevant
+## Cover letter
+
+Describe in plain language the user visible changes of this feature.
+
+**Issue(s)**: [LIST OF RELEVANT ISSUES HERE]
 
 When referencing a related issue, remember to migrate duplicate stories from the
 external tracker. This is not relevant for most users.
```

Comment:
I'd vote for removing these lines too (since as they aren't relevant for most people).

Reply:
+1